modules/flowable-task-service/src/main/java/org/flowable/task/service/impl/HistoricTaskServiceImpl.java

```diff
@@ -79,15 +79,15 @@ public void insertHistoricTask(HistoricTaskInstanceEntity historicTaskInstanceEn
     public void deleteHistoricTask(HistoricTaskInstanceEntity HistoricTaskInstance) {
         getHistoricTaskInstanceEntityManager().delete(HistoricTaskInstance);
     }
-    
+
     @Override
     public HistoricTaskInstanceEntity recordTaskCreated(TaskEntity task) {
         HistoricTaskInstanceEntityManager historicTaskInstanceEntityManager = getHistoricTaskInstanceEntityManager();
-        HistoricTaskInstanceEntity historicTaskInstanceEntity = historicTaskInstanceEntityManager.create(task); 
-        historicTaskInstanceEntityManager.insert(historicTaskInstanceEntity, false);
+        HistoricTaskInstanceEntity historicTaskInstanceEntity = historicTaskInstanceEntityManager.create(task);
+        historicTaskInstanceEntityManager.insert(historicTaskInstanceEntity, true);
```

Comment:
The problem is to keep events in sync with 
org.flowable.engine.impl.history.async.json.transformer.TaskCreatedHistoryJsonTransformer#transformJson
which uses
            historicTaskService.insertHistoricTask(historicTaskInstance, true); // fire events